resources/examples/minimal.m4

```diff
@@ -3,26 +3,20 @@
 # m4_ignore(
 echo "This is just a script template, not the script (yet) - pass it to 'argbash' to fix this." >&2
 exit 11  #)Created by argbash-init v2.10.0
-# ARG_OPTIONAL_SINGLE([option], o, [A option with short and long flags and default], [boo])
-# ARG_OPTIONAL_BOOLEAN([print], , [A boolean option with long flag (and implicit default: off)])
-# ARG_POSITIONAL_SINGLE([positional-arg], [Positional arg description], )
+# ARG_OPTIONAL_SINGLE([option])
+# ARG_OPTIONAL_BOOLEAN([print])
+# ARG_POSITIONAL_SINGLE([positional-arg])
```

Comment:
This looks incorrect - generation of this file has two phases, when the first phase is what you have committed, and the second phase patches this and adds those descriptions. Try to go to `doc` and run `make html` there - that should do the patch.